### ISSUE TITLE
output 36 (simulatedEIR) per vector species?

### ISSUE THREAD

**Issue by a-e-e-m** [SwissTPH/openmalaria#232]
For my purposes it would be very interesting to have the output simulatedEIR per vector species. The second column is currently not used for this output measure. Is this doable?
best wishes,
Adrian

**Comment by dhardy** [comment]
Time-step EIR is calculated as follows:
```
TS_EIR = adult_innoculations / num_adults
adult_innoculations = sum_adults[ sum_vectors( EIR ) ]
```
So yes, we could. Currently we report `innoculationsPerAgeGroup` which is categorised by age-group, cohort and genotype, but not vector species.

Our reporting system can handle classification by vector species and optionally genotype, but not simultaneously with age-group and cohort.

@ThomasASmith has the final say on things like this.

**Comment by ThomasASmith** [comment]
Sorry not to get back sooner.  I wasn't sure who is  @a-e-e-m or why it is requested, but now I understand.
If it is not a big deal then I can approve it.

Tom

**Comment by dhardy** [comment]
Appologies @a-e-e-m for taking so long to get to this.

I think the best option is to create a new output, `innoculationsPerVector`, so as not to interfere with existing analysis done on `simulatedEIR`.

**Comment by dhardy** [comment]
Implemented by #238. [Release 40.3](https://github.com/SwissTPH/openmalaria/releases/tag/schema-40.3) contains the [new measure](https://github.com/SwissTPH/openmalaria/wiki/XmlMonitoring#eir-per-species).
